doc/api/cli.md

```diff
@@ -1692,6 +1697,11 @@ is being linked to Node.js. Sharing the OpenSSL configuration may have unwanted
 implications and it is recommended to use a configuration section specific to
 Node.js which is `nodejs_conf` and is default when this option is not used.
 
+### `--env-file-optional=config`
```

Comment:
IMO `--env-file-if-exists` is a clearer name.
```suggestion
### `--env-file-if-exists=config`
```

Reply:
Yup, currently working on it. Haven't pushed changes yet, only updated branch with upstream changes 👍🏼